bigbluebutton-html5/imports/ui/components/chat/chat-graphql/chat-message-list/page/chat-message/message-content/presentation-content/component.tsx

```diff
@@ -49,6 +49,13 @@ const ChatMessagePresentationContent: React.FC<ChatMessagePresentationContentPro
   assertAsMetadata(presentationData);
 
   const downloadUrl = `${APP_CONFIG.bbbWebBase}/${presentationData.fileURI}`;
+  const parseFilename = (filename = '') => {
```

Comment:
the name of the function should be something like `getFilenameWithoutExtension` or `trimExtension`.
the name `parseFilename` does not say what the function does.